README.md

```diff
@@ -13,6 +13,25 @@
   - [License](#license)
 
 ## Usage
+Credentials must be exported to the CLI prior to getting started.
+
+```sh
+
+export AWS_ACCESS_KEY_ID=... 
+export AWS_SECRET_ACCESS_KEY=...
+export AWS_SESSION_TOKEN=...
+
+```
+
+After deploying the mesh, deploy HashiCups demo app. 
+
+Demo can be found here: https://github.com/hashicorp-education/learn-consul-get-started-kubernetes/tree/main/self-managed/eks/hashicups 
```

Comment:
I think the idea here is to show folks how they can deploy hashicups here, so creating a folder here and copying yaml files would be better, please make sure the deployment actually works. thanks!

Reply:
Suggested edits have been made.